assets/js/amber.js

```diff
@@ -226,3 +226,18 @@ export class Socket {
 module.exports = {
   Socket: Socket
 }
+
+
+/**
+ * Allows delete links to post for security and ease of use similar to Rails jquery_ujs
+ */
+$(document).ready(() => {
+  $( "a[data-method='delete']" ).click(function(){
+    if(confirm("Are you sure you want to delete this?")){
+      const form = $(`<form action='${this.href}' method='POST'><input type='hidden' name='_method' value='DELETE' /></form>`);
+      $(document.body).append(form);
+      form.submit();
+    }
```

Comment:
Also there seems to be missing Ajax error handling to this method.

Reply:
This isn't an Ajax submission. It actually posts like with rails ujs